script.js

```diff
@@ -0,0 +1,77 @@
+// Write your solutions below
+
+// Go through the items and find all results that have kind of shopping#product. 
+// Print the count of these results. 
+
+var shop = products.items;
+var count = 0;
+
+for (i=0; i<shop.length; i++) {
+    if (shop[i].kind == "shopping#product") {
+        count++;
+    }
+}
+
+console.log(count);
+
+// Where else is this count information stored in the search results?
+console.log("I have no idea.");
+
+
+// Print the title of all items with a backorder availability in inventories.
+var backOrders = [];
+

```

Comment:
I dont think you need to nest 3 loops.

Try to look at your classmates pull request. I think one loop will suffice.